packages/vsce/src/webviews/resource-inspector-panel/ResourceInspector.tsx

```diff
@@ -35,105 +36,193 @@ const ResourceInspector = () => {
           {resourceInfo?.highlights.length > 0 && (
-            <tr>
-              <p className="padding-top-30"></p>
+            <tr className="resource-info-rows">
               {resourceInfo.highlights.map((highlight) => (
-                <p className="line padding-left-27">
-                  {highlight.key}: {highlight.value}
-                </p>
+                <td key={highlight.key} className="resource-info-row">
+                  <span className="vscode-breadcrumb-foreground-color">{highlight.key}:</span> <span className="vscode-badge-foreground-color">{highlight.value}</span>
+                </td>
               ))}
             </tr>
           )}
         </tbody>
       </table>
-
       <table className="border-collapse">
-        <thead id="table-header-2" className="thead-2 vertical-align-sub">
-          <th className="div-display-1 th-2">Attribute</th>
-          <th className="padding-right-10 th-3">
-            <div>
-              <div className="div-display-1 vertical-align-sub">Value</div>
-              <VscodeTextfield
-                type="text"
-                placeholder="Keyword search..."
-                onInput={(e: { target: HTMLInputElement; }) => setSearch(e.target.value)}
-                value={search}
-                className="search-style div-display-1"
-              ></VscodeTextfield>
-            </div>
-          </th>
+        <thead id="attributes-header" className="attributes-header-section">
+          <tr>
+            <th className="attributes-title" colSpan={2}>
+              <div className="attributes-header-row">
+                <div className="header-label-div">
+                  <span className="header-label">ATTRIBUTE</span>
+                </div>
+                <div className="header-label-value-div">
+                  <span className="header-label-value">VALUE</span>
+                </div>
+                <VscodeTextfield
+                  type="text"
+                  placeholder="Keyword search..."
+                  onInput={(e: { target: HTMLInputElement }) => setSearch(e.target.value)}
+                  value={search}
+                  className="attribute-search"
+                />
+              </div>
+            </th>
+          </tr>
         </thead>
         <tbody>
-          {resourceInfo && Object.entries(resourceInfo.resource)
-            .filter(([key, value]) => !key.startsWith("_"))
-            .filter(
-              ([key, value]) =>
-                key.toLowerCase().trim().includes(search.toLowerCase().trim()) || value.toLowerCase().trim().includes(search.toLowerCase().trim())
-            )
-            .map(([key, value]) => (
-              <tr key={key}>
-                <td className="padding-left-27 width-30">{key}</td>
-                <td className="padding-right-75 width-70">{value}</td>
-              </tr>
-            ))}
+          {resourceInfo &&
+            Object.entries(resourceInfo.resource)
```

Comment:
```suggestion
          {resourceInfo && Object.entries(resourceInfo.resource)
```
i think this was clear on one line